src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqResponsibilitiesRepository.kt

```diff
@@ -123,6 +123,25 @@ class JooqResponsibilitiesRepository(
         }
     }
 
+    override fun getResponsibilityId(groupId: String, touchstoneVersionId: String, scenarioId: String): Int
+    {
+        val responsibilitySet = getResponsibilitySet(groupId, touchstoneVersionId)
```

Comment:
Should throw errors if group or touchstone don't exist?

Reply:
Actually that logic should now live in the logic class! But yes you're right, I'll add it there :+1: